scenarios/test_scenarios/test_pv.osm

```diff
@@ -0,0 +1,42 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
There's already `gs_straight_pv.osm`, why adding another one?

Reply:
Better to add another path vehicle to the existing scenario to test different settings.